core/src/main/java/ar/com/wolox/wolmo/core/util/NavigationUtils.java

```diff
@@ -116,39 +195,48 @@ public Builder addSharedElement(View sharedView, String sharedString) {
             return this;
         }
 
-        public Builder addIntentObjects(IntentObject... intentObjects) {
-            for (IntentObject intentObject : intentObjects) addIntentObject(intentObject);
+        public Builder addIntentObjects(IntentExtra... intentExtras) {
+            for (IntentExtra intentExtra : intentExtras) addIntentObject(intentExtra);
```

Comment:
I think we should add curly braces to the `for`. For me it's OK to use inline `if`, but for the loops I think it's better to use braces. You can keep it like this if you want. :smile: